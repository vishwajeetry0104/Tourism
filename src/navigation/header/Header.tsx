import {SafeAreaView, StyleSheet, View} from 'react-native';
import Aloha from '../../assets/svg/Aloha.svg';
import * as React from 'react';

const styles = StyleSheet.create({
  headerContainer: {backgroundColor: '#FFFFFF'},
  headerLogoContainer: {
    padding: 20,
    justifyContent: 'center',
    alignItems: 'center',
  },
});

function LogoHeader() {
  return (
    <SafeAreaView style={styles.headerContainer}>
      <View style={styles.headerLogoContainer}>
        <Aloha width={94} height={35} />
      </View>
    </SafeAreaView>
  );
}

export default LogoHeader;
