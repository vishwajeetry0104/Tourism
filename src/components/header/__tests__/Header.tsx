import React from 'react';
import {act, create} from 'react-test-renderer';
import Header from '../Header';

describe('Header', () => {
  const createWrapper = (updatedProps = {}) => {
    return create(<Header {...updatedProps} />);
  };

  let defaultWrapper;

  act(() => {
    defaultWrapper = createWrapper();
  });

  afterAll(() => {
    defaultWrapper.unmount();
  });

  it('should render Header component without issues', () => {
    expect(defaultWrapper).toMatchSnapshot();
  });
});
