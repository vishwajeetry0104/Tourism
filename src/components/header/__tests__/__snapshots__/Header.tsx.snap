// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render Header component without issues 1`] = `
<RCTSafeAreaView
  style={
    {
      "backgroundColor": "#FFFFFF",
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "justifyContent": "center",
        "padding": 20,
      }
    }
  >
    <SvgMock
      height={35}
      width={94}
    />
  </View>
</RCTSafeAreaView>
`;
